dev/tools/merge-pr.sh

```diff
@@ -77,11 +78,13 @@ if [ -z "$REMOTE" ]; then
   error "please run: git branch --set-upstream-to=THE_REMOTE/$CURRENT_LOCAL_BRANCH"
   exit 1
 fi
-REMOTE_URL=$(git remote get-url "$REMOTE" --push)
-if [ "$REMOTE_URL" != "$OFFICIAL_REMOTE_URL" -a \
-     "$REMOTE_URL" != "$OFFICIAL_REMOTE_URL.git" ]; then
+REMOTE_URL=$(git remote get-url "$REMOTE" --all)
+if [ "$REMOTE_URL" != "$OFFICIAL_REMOTE_GIT_URL" -a \
+     "$REMOTE_URL" != "$OFFICIAL_REMOTE_GIT_URL.git" -a \
+     "$REMOTE_URL" != "$OFFICIAL_REMOTE_HTTPS_URL" -a \
```

Comment:
Please don't use `-a` as bash can parse it in unintended ways.

Reply:
OK. Anyways, it is going to conflict with the shell script PR. Is it ready to be merged? If yes, I'll rebase afterwards.